src/sass/components/_customersreview.scss

```diff
@@ -0,0 +1,64 @@
+.customers-img {
+  padding: 60px 97px 32px 98px;
+}
+
+.customers-first-box {
+  background-color: #fafafa;
+  position: relative;
+  z-index: -10;
+  &__gradient {
+    background-image: repeating-linear-gradient(
+      rgba(0, 0, 0, 0) 0,
+      rgba(0, 0, 0, 0) 29px,
+      #e1e1e1 29px,
+      rgba(0, 0, 0, 0) 30px
+    );
+  }
+  &__icon {
```

Comment:
Cudzysłów trochę za mało zachodzi za napisy, dałoby radę go przesunąć trochę? :)

Reply:
dać się da na pewno :D, ale mówiłem Ci nawet wtyczkę po to specjalnie pobrałem :P